src/kili/use_cases/issue/__init__.py

```diff
@@ -4,17 +4,20 @@
 
 from kili.adapters.kili_api_gateway import KiliAPIGateway
 from kili.adapters.kili_api_gateway.issue.types import IssueToCreateKiliAPIGatewayInput
+from kili.domain.issue import IssueId
 from kili.entrypoints.mutations.issue.helpers import get_labels_asset_ids_map
 from kili.use_cases.issue.types import IssueToCreateUseCaseInput
 
 
 class IssueUseCases:
     """Issue use cases."""
 
-    def __init__(self, kili_api_gateway: KiliAPIGateway):
+    def __init__(self, kili_api_gateway: KiliAPIGateway) -> None:
         self._kili_api_gateway = kili_api_gateway
 
-    def create_issues(self, project_id, issues: List[IssueToCreateUseCaseInput]):
+    def create_issues(
+        self, project_id: str, issues: List[IssueToCreateUseCaseInput]
+    ) -> List[IssueId]:
```

Comment:
It is better to force the return type hint even if it is well infered from the function implem with inputs ?

Reply:
Doesn't cost much to add it. I find that it helps, especially during refactoring